src/rundeck/templates/configmap.yaml

```diff
@@ -0,0 +1,154 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ template "rundeck.name" . }}
+    chart: {{ template "rundeck.chart" . }}
+    component: "{{ template "rundeck.name" . }}"
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "rundeck.fullname" . }}
+data:
+  framework.properties: |-
+    #    
+    # framework.properties -
+    #
+    
+    # ----------------------------------------------------------------
+    # Rundeck server connection information
+    # ----------------------------------------------------------------
+    
+    framework.server.name = localhost
+    framework.server.hostname = localhost
+    framework.server.port = 4440
+    framework.server.url = http://localhost:4440
+    
+    # ----------------------------------------------------------------
+    # Installation locations
+    # ----------------------------------------------------------------
+    
+    rdeck.base={{ .Values.env.RUNDECK_INSTALL_DIR }}
+    
+    framework.projects.dir={{ .Values.env.SHARED_STORAGE }}/projects
+    framework.etc.dir={{ .Values.env.RUNDECK_INSTALL_DIR }}/etc
+    framework.var.dir={{ .Values.env.RUNDECK_INSTALL_DIR }}/var
+    framework.tmp.dir={{ .Values.env.RUNDECK_INSTALL_DIR }}/var/tmp
+    framework.logs.dir={{ .Values.env.SHARED_STORAGE }}/logs
+    framework.libext.dir={{ .Values.env.RUNDECK_INSTALL_DIR }}/libext
+    
+    # ----------------------------------------------------------------
+    # SSH defaults for node executor and file copier
+    # ----------------------------------------------------------------
+    
+    framework.ssh.keypath = {{ .Values.env.RSA_KEY_DIR }}/id_rsa
+    framework.ssh.user = {{ .Values.env.RD_USER }}
+    
+    # ssh connection timeout after a specified number of milliseconds.
+    # "0" value means wait forever.
+    framework.ssh.timeout = 0
+    
+    # ----------------------------------------------------------------
+    # Auto generated server UUID
+    # ----------------------------------------------------------------
+    rundeck.server.uuid =

```

Comment:
In the prior PR you said: "uuid generation is at dockerfile and shell script level. so not present in the chart"

I don't see the Dockerfile anywhere.

Reply:
Reference of same is there in Chart.yaml
source:
  - https://github.com/Deepak-Routray/rundeck-docker

Relevant Changes here:
https://github.com/Deepak-Routray/rundeck-docker/blob/master/data/bootstrap.sh#L63